models/userProfile.js

```diff
@@ -0,0 +1,28 @@
+const database = require('../utils/database');
+
+
+exports.GetUserProfile = function (listingPK, callback = (result) => { }) 
+{
+	var db = database.connectDatabase();
+	var query = `
+    SELECT
+	    US_PK,
+	    US_Username,
+	    US_Password,
+    	US_Email,
+	    US_FirstName,
+	    US_LastName,
+	    US_PhoneNumber,
+	    US_BirthDate,
+    FROM User 
+    WHERE US_Username = ?
```

Comment:
also, I'm not sure how you're currently doing it, but it would be preferred if you could make the api get the username from the session, that way, by using localhost:3000/user/profile, we can show each user their own page, based on them being logged in.

Also, consider what should happen if theyre not logged in (probably just show them a json.failResponse for now)

Reply:
Sorry, I didn't think of that, I'll change it so it gets the username from the session